project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/TestOverriddenDeleteAction.kt

```diff
@@ -0,0 +1,19 @@
+package org.plan.research.minimization.plugin
+
+import org.plan.research.minimization.plugin.services.TestActionService
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.components.service
+import mu.KotlinLogging
+
+class TestOverriddenDeleteAction : AnAction() {
```

Comment:
That's a concept I was thinking of. The concept is basically to implement some test actions with only a part of a functionality (e.g. fetching the overridden methods) to manually test a feature without actually implementing all the steps. 

However, as you can see I haven't used it as much as I would have (e.g. unit tests) because 
1. Test on a full-scale project is hard and takes a lot of time
2. Writing small projects is complicated too
3. Maybe I will make integration tests possible and that will be no necessary

But I left it to discuss it with you and because I could forget it.